packages/api/src/endpoints/channel/HelixAdSchedule.ts

```diff
@@ -0,0 +1,53 @@
+// import { Enumerable } from '@d-fischer/shared-utils';
+import { DataObject, rawDataSymbol, rtfm } from '@twurple/common';
+// import type { BaseApiClient } from '../../client/BaseApiClient';
+import type { HelixAdScheduleData } from '../../interfaces/endpoints/channel.external';
+// import type { HelixUser } from '../user/HelixUser';
+
+/**
+ * Represents a broadcaster's ad schedule.
+ */
+@rtfm('api', 'HelixAdSchedule')
+export class HelixAdSchedule extends DataObject<HelixAdScheduleData> {
+	/**
+	 * The number of snoozes available for the broadcaster.
+	 */
+	get snoozeCount(): number {
+		return this[rawDataSymbol].snooze_count;
+	}
+
+	/**
+	 * The UTC Unix Epoch timestamp when the broadcaster will gain an additional snooze.
```

Comment:
We don't usually expose timestamps like that, please wrap all timestamps in `Date` objects and change the property names for these accordingly (we generally use a `Date` suffix) for these